<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Smart Matrix</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/icon/chip.png</normaloff>:/icon/chip.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="MessageBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>显示原数据</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>忽略数据帧</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="SaveCheckBox">
            <property name="text">
             <string>保存为文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="MessageEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>选择设备：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string/>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SendButton">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>总帧数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>已发送：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>已接收：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>已丢弃：
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="graphicsView_2"/>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>开启绘图</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>显示压力</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>FPS: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>连接</string>
    </property>
    <addaction name="connectSerialAct"/>
    <addaction name="connectBlueToothAct"/>
    <addaction name="connectUSBAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>命令</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>串口</string>
     </property>
     <addaction name="beginSerialAct"/>
     <addaction name="stopSerialAct"/>
    </widget>
    <widget class="QMenu" name="menuUSB3_0">
     <property name="title">
      <string>USB3.0</string>
     </property>
     <addaction name="action_11"/>
     <addaction name="action_15"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>蓝牙</string>
     </property>
     <addaction name="action_12"/>
     <addaction name="action_17"/>
    </widget>
    <widget class="QMenu" name="menuSD">
     <property name="title">
      <string>SD卡</string>
     </property>
     <addaction name="action_13"/>
     <addaction name="action_16"/>
     <addaction name="action_18"/>
     <addaction name="action_19"/>
     <addaction name="action_20"/>
    </widget>
    <addaction name="stopAct"/>
    <addaction name="menu_5"/>
    <addaction name="menuUSB3_0"/>
    <addaction name="menu_6"/>
    <addaction name="menuSD"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>绘图</string>
    </property>
    <addaction name="action_27"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_21"/>
    <addaction name="action_22"/>
    <addaction name="action_23"/>
    <addaction name="action_24"/>
    <addaction name="action_25"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="savePositonAct"/>
   </widget>
   <addaction name="menu_7"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="connectSerialAct">
   <property name="text">
    <string>串口</string>
   </property>
  </action>
  <action name="connectBlueToothAct">
   <property name="text">
    <string>蓝牙</string>
   </property>
  </action>
  <action name="connectUSBAct">
   <property name="text">
    <string>USB3.0</string>
   </property>
  </action>
  <action name="stopAct">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="beginSerialAct">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="stopSerialAct">
   <property name="text">
    <string>结束</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>结束</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>结束</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>结束</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>新卡</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>初始</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>获取配置</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>速率</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>阈值</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>关灯</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>开灯</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>矩阵规模</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>分配设备号</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>保存至</string>
   </property>
  </action>
  <action name="savePositonAct">
   <property name="text">
    <string>保存位置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
